cmd/create/decision/cmd.go

```diff
@@ -38,7 +38,6 @@ func NewCreateDecisionCommand() *cobra.Command {
 	options := NewDecisionOptions()
 	cmd := &cobra.Command{
 		Use:     use,
-		Aliases: []string{"access-request"},
```

Comment:
Out of curiosity - why are we removing this alias?

Reply:
it's 'decision', not 'access-request'